ui/packages/ui/src/main/webapp/js/models/installer/Configuration.js

```diff
@@ -79,12 +79,6 @@ define([
                 propertiesMap[model.get('key')] = model.get('value');
             });
 
-            var devMode = location.search.indexOf('dev=true') > -1 ? true : false;
```

Comment:
I've reverted this (https://github.com/codice/ddf/pull/3841) as it's easier to test the PR without this part, but I'm curious what people think the behavior should be if ?dev=true.

Reply:
As long as you can still use the installer UI with `?dev=true`, I think it's fine to revert https://github.com/codice/ddf/pull/3841.